alertproduction.tex

```diff
@@ -1,11 +1,39 @@
+
 % Alert Production in year 1
 % Responsible: Eric Bellm
-\section{Alert Production in Year One}
+\section{Alert Production in Commissioning and Early Operations}
 \label{sec:pp}
 
-{\textit Describe the relaxed requirements on noise contributed to a difference image by the template 
+\subsection{Processing Overview}
+
+The \DPDD{} summarizes the pipelines which will be used during Prompt Processing to produce alerts as well as other prompt data products (\S \ref{sec:data}), including Solar System Processing.
+In brief, raw images have instrument signatures removed and are photometrically and astrometrically calibrated.
+When template images for the corresponding region of the sky are available, the template is subtracted from the new processed visit image and sources are detected on the image difference.
+Alerts are then generated for all DIASources detected at five sigma in the difference.
+At the end of the night, DIASources without a history of previous detection are input into Solar System Processing, which attempts to link them with other past detections and identify new Solar System objects.
+
+Both Alert Production and Solar System Processing thus depend on the presence of template images.
+During steady-state operations, these templates will be constructed during the annual Data Releases and will built from the best available subset of images taken.
+The input images for DRP-produced templates will accordingly have very good seeing and  comprehensive spatial coverage.
+Coadding multiple images enables artifact rejection  \citedsp{DMTN-080} and reduces noise.
+All of these template characteristics all help to ensure that image differencing is highly complete and highly pure.
+
+To enable alert production to proceed during commissioning and early operations, it is necessary to accept templates of lower quality.
+Because we have a smaller set of input images to choose from and uncertain knowledge about future observations, on-the-fly (or incremental) template generation necessarily must balance the trade off of earlier template availibility against template quality and spatial completeness. 
```

Comment:
I wonder of we should use this document to drop "incremental template generation". My original idea behind the term was to incrementally build a template library duuring the course on year one. This is not how it is understood though and I think leads to confusion.

Reply:
Yes, let's discuss this further--I'm not immediately sure of a better candidate but I can see how it is confusing.